manifests/scripts/metrics-dashboard/prometheus/openshift-operators-service-monitor.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
Shouldn't the service monitor be for the gitops-service-operator rather than the gitops-operator?  Or am I totally confuse about what we're doing?